docs/src/prototypes/VirtualizedTable/VirtualizedTable.tsx

```diff
@@ -0,0 +1,80 @@
+import {
+  gridCellBehavior,
+  gridHeaderCellBehavior,
+  gridNestedBehavior,
+  gridRowBehavior,
+  Table,
+} from '@fluentui/react'
+import * as React from 'react'
+import { AutoSizer, List as ReactVirtualizedList } from 'react-virtualized'
+import getItems from './itemsGenerator'
+
+const { rows } = getItems()
+
+const VirtualizedTablePrototype = () => {
+  const rowGetter = ({ index }) => {
+    return rows[index]
+  }
+
+  const accessibilityListProperties = {
+    'aria-label': '',
+    'aria-readonly': undefined,
+    containerRole: 'presentation',
+    role: 'presentation',
+  }
+
+  const rowRenderer = ({ index, style }) => {
+    const row = rows[index]
+    const topOffset = `${style.top}px`
+    const leftOffset = `${style.left}px`
+    const height = `${style.height}px`
+    return (
+      <Table.Row
+        design={{
+          top: topOffset,
+          left: leftOffset,
+          width: style.width,
+          height,
+          position: style.position,
+        }}
+        key={row.key}
+        accessibility={gridRowBehavior}
+        aria-rowindex={index + 1}
+      >
+        <Table.Cell {...row.items[0]} accessibility={gridCellBehavior} />
+        <Table.Cell {...row.items[1]} accessibility={gridCellBehavior} />
+        <Table.Cell {...row.items[2]} accessibility={gridCellBehavior} />
+        <Table.Cell {...row.items[3]} accessibility={gridCellBehavior} />
+      </Table.Row>
+    )
+  }
+
+  return (
+    <AutoSizer disableHeight>
+      {({ width }) => (
+        <Table accessibility={gridNestedBehavior} aria-rowcount={rows.length}>
+          <Table.Row header accessibility={gridRowBehavior} styles={{ width: `${width}px` }}>
```

Comment:
```suggestion
          <Table.Row header accessibility={gridRowBehavior} style={{ width }}>
```

If we will use `styles` the passed rules will be processed by Fela and this has some caveats in this case:
- it will generate a new atomic class name for each value, i.e. (`.a { width: 400px }, .b { width: 401px }`)
- this can degrade performance as it will disable caching in future